src/main/resources/application.yml

```diff
@@ -15,7 +15,7 @@ spring:
     driver-class-name: org.postgresql.Driver
   jpa:
     hibernate:
-      ddl-auto: update
+      ddl-auto: create
```

Comment:
if you change the update to create - change it back, or don't add these changes to commit.

Reply:
ok, fixed